DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RawEventData.h

```diff
@@ -0,0 +1,149 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+//file RawEventData.h class  for RAW data format
+//Alla.Maevskaya@cern.ch
+// with Artur.Furs
+//
+#ifndef ALICEO2_FIT_RAWEVENTDATA_H_
+#define ALICEO2_FIT_RAWEVENTDATA_H_
+
+#include "DataFormatsFT0/Digit.h"
+#include "Headers/RAWDataHeader.h"
+#include "DataFormatsFT0/LookUpTable.h"
+#include <CommonDataFormat/InteractionRecord.h>
+#include <iostream>
+#include <bitset>
+#include "Rtypes.h"
+namespace o2
+{
+namespace ft0
+{
+constexpr int NCHANNELS_FT0 = 208;
+constexpr int NCHANNELS_PM = 12;
+constexpr int NPMs = 18;
+constexpr int NBITS_EVENTDATA = 9;
+
+struct EventHeader {
+  ushort startDescriptor : 4;
+  uint reservedField : 28;
+  uint orbit : 32;
+  short bc : 12;
+  short nGBTWords : 4;
+  ClassDefNV(EventHeader, 1);
+};
```

Comment:
sizeof(EventHeader) is 12B. If you consider option of writing w/o padding, you should be able to write only 10B. Alternatively, you should make sure that highest 2 bytes are set to 0. Also, it is better to use for the BC the same fields as in the int. record. So, you better use:
````
struct EventHeader {
  constexpt int PayloadSize = 10; // size in bytes = 1GBT word
  union {
    uint32_t w[3] = {0};
    struct {  
      uint8_t startDescriptor : 4;
      uint32_t reservedField : 28;
      uint32_t orbit : 32;
      uint16_t bc : 12;
      uint8_t nGBTWords : 4;
    };
  };
  ClassDefNV(EventHeader, 1);
};
````